packages/ripple-nuxt-tide/modules/landing-page/components/TideOpenForm.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="tide-open-form">
+    <a class="openforms-embed tide-open-form__link" :href="formLink">Click here to view form.</a>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'TideOpenForm',
+  props: {
+    formLink: { default: null, type: String }
+  },
+  mounted: function () {
+    let openFormsEmbedScript = document.createElement('script')
+    openFormsEmbedScript.setAttribute('src', 'https://au.openforms.com/Scripts/embed-iframe.js?immediate=true')
+    document.head.appendChild(openFormsEmbedScript)
+  }
+}
+</script>
+
+<style lang="scss">
+@import "~@dpc-sdp/ripple-global/scss/settings";
+
+.tide-open-form {
+  // This OpenForm component will cause high CLS(cumulative layout shift) issue.
```

Comment:
Nice, how to know this thing? I should consider this as well when updating/building new feature.

Reply:
@rubyllamadora Only happens when a client JS changes the layout and we should avoid it if possible. As this is in OpenForms so I can't do too much. You can install https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma?hl=en to measure it.

Currently the alerts on page top cause high CLS issue on mobile which we need to fix it as well.